### ISSUE TITLE
If another table sorting library is being used don't apply table-sort-js (browser extensions)

### ISSUE THREAD

**Issue by LeeWannacott** [LeeWannacott/table-sort-js#102]
https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

This table is already using `sortable` and `jquery-tablesorter` classes we should check for sorting classes from other libraries before running  table sort js. If they exist don't run table-sort-js...

**Comment by LeeWannacott** [comment]
Or do we be evil and remove other libraries table class tags :supervillain:

**Comment by scodes73** [comment]
> Or do we be evil and remove other libraries table class tags supervillain

How about we give a toggle for user to pick it?

**Comment by LeeWannacott** [comment]
@scodes73 Yeah that sounds good; it's what I was thinking. Default don't run if other sorting libraries would be helpful for the browser plugins (which currently have no configurability by users) and then a class for overriding libraries if they exist:  evil-supervillian-mode!
